intTests/test_solver_cache/test.sh

```diff
@@ -0,0 +1,28 @@
+set -e
+
+# Testing the basic features of the solver cache
+SAW_SOLVER_CACHE_PATH="test_solver_cache.cache" $SAW test_basics.saw
+
+# Testing the `set_solver_cache_path` command as well as re-using a cache file
+$SAW test_path_and_reuse.saw
+
+# Testing cleaning the solver cache
+pip install cbor2 python-dateutil lmdb
+python3 -i ../../saw-remote-api/python/saw_client/solver_cache.py << END
```

Comment:
Does this test still need to rely on Python to work, or can it be implemented in terms of SAWScript? If it _does_ rely on Python, perhaps we should move this test to the test suite for the Python bindings.

Reply:
Good point! It does not rely on the remote API – the only reason this bit calls out to Python is so that we can manually mess with the version strings of certain entries to make sure the `clean_solver_cache` command works. I removed the dependency on the remote API (instead just using the Python LMDB bindings directly) and added a comment explaining what this bit of Python code does.